src/components/ReportActionItem/MoneyRequestView.js

```diff
@@ -225,6 +230,16 @@ function MoneyRequestView({report, betas, parentReport, policyCategories, should
                     />
                 </OfflineWithFeedback>
             )}
+            {shouldShowBillable && (
+                <View style={[styles.flexRow, styles.mb4, styles.justifyContentBetween, styles.alignItemsCenter, styles.ml5, styles.mr8]}>
+                    <Text color={!transactionBillable ? themeColors.textSupporting : undefined}>{translate('common.billable')}</Text>
+                    <Switch
+                        accessibilityLabel={translate('common.billable')}
+                        isOn={transactionBillable}
+                        onToggle={(value) => IOU.editMoneyRequest(transaction.transactionID, report.reportID, {billable: value})}
```

Comment:
At the time of writing this code. `IOU.editMoneyRequest` was only used for regular money requests. If we call this function on a distance money request it won't work as expected (regression: https://github.com/Expensify/App/issues/31128). There was another function `IOU.updateDistanceRequest` that we should have called in that case.

PS: Now `IOU.editMoneyRequest` is unified to handle both regular and distance money requests.